src/components/toolbar.test.tsx

```diff
@@ -6,10 +6,11 @@ import { ModalProvider } from "react-modal-hook";
 import { DocumentModel } from "../models/document/document";
 import { DocumentContentModel } from "../models/document/document-content";
 import { createStores } from "../models/stores/stores";
-import { ToolbarComponent, ToolbarConfig } from "./toolbar";
+import { ToolbarComponent } from "./toolbar";
 
 // This is needed so MST can deserialize snapshots referring to tools
 import "../register-tools";
+import { ToolbarModel, ToolbarModelSnapshot } from "../models/stores/app-config-model";
```

Comment:
I would move this up with the `ToolbarComponent` import, but it's test code so 🤷 .